README.md

```diff
@@ -4,12 +4,13 @@ This repository contains example plugins to showcase different use cases.
 
 ## App plugins
 
-| Example                                             | Description                                                               |
-| --------------------------------------------------- | ------------------------------------------------------------------------- |
-| [app-basic](examples/app-basic)                     | demonstrates how to build a basic app plugin that uses custom routing.    |
-| [app-with-dashboards](examples/app-with-dashboards) | demonstrates how to include pre-built dashboards in an app plugin.        |
-| [app-with-backend](examples/app-with-backend)       | demonstrates how to build an app plugin with its own backend.             |
-| [app-with-extensions](examples/app-with-extensions) | demonstrates how to build an app plugin that extends the Grafana core ui. |
+| Example                                                       | Description                                                                                       |
+| ------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| [app-basic](examples/app-basic)                               | demonstrates how to build a basic app plugin that uses custom routing.                            |
+| [app-with-dashboards](examples/app-with-dashboards)           | demonstrates how to include pre-built dashboards in an app plugin.                                |
+| [app-with-backend](examples/app-with-backend)                 | demonstrates how to build an app plugin with its own backend.                                     |
+| [app-with-extensions](examples/app-with-extensions)           | demonstrates how to build an app plugin that extends the Grafana core ui.                         |
+| [app-with-extension-point](examples/app-with-extension-point) | demonstrates how to add an extension point in the plugin UI that can be extended by other plugins |
```

Comment:
**Nit:** should we maybe call this placement (to be coherent with how we call these in other places)?
```suggestion
| [app-with-extension-point](examples/app-with-extension-point) | demonstrates how to add an extension placement in the plugin UI that can be extended by other plugins |
```